features/MAT001_Material-set.feature

```diff
@@ -0,0 +1,97 @@
+@implementer-agreement
+@MAT
+@version1
+@E00020
+Feature: MAT001 - Material Set
+  The rule verifies that correct names are used for material sets.
+
+  Scenario Outline: Verify material names for entities
+    Given an <entity>
+    Given its attribute HasAssociations
+    Given its attribute RelatingMaterial
+    Given Each relating material layer
```

Comment:
```suggestion
    Given Each related material component
```

I'm struggling to find the correct word, but layer is too specific. Because it also involves constituents or profiles.